Sources/SwiftSyntax/SyntaxText.swift

```diff
@@ -142,19 +147,29 @@ public struct SyntaxText {
 
 /// `SyntaxText` is a collection of `UInt8`.
 extension SyntaxText: RandomAccessCollection {
+  /// SyntaxText operates on bytes and each byte is represented by a `UInt8`.
   public typealias Element = UInt8
+
+  /// `SyntaxText` is a continuous memory region that can be accessed by an integer.
   public typealias Index = Int
+
+  /// `Slice<SyntaxText>` represents a part of a ``SyntaxText``.
   public typealias SubSequence = Slice<SyntaxText>
 
+  /// The index of the first byte in ``SyntaxText``
   public var startIndex: Index { buffer.startIndex }
+
+  /// The index one after the last byte in ``SyntaxText``.
   public var endIndex: Index { buffer.endIndex }
 
-  public subscript(position: Index) -> Element {
-    get { return buffer[position] }
+  /// Access the byte at `index`.
+  public subscript(index: Index) -> Element {
+    get { return buffer[index] }
   }
 }
 
 extension SyntaxText: Hashable {
+  /// Returns `true` iff `lhs` and `rhs` contain the same bytes.
```

Comment:
Double `f` in if.

Reply:
Actually, I meant `iff` here (https://en.wikipedia.org/wiki/If_and_only_if). It’s a shorthand of “if and only if”. But then I remembered that sometime in the past I actually changed all the “iff” to “if” because “iff” looks too much like a typo most of the time. Thanks for reminding me that “iff” is not a standard english word 😉